src/App.js

```diff
@@ -1,24 +1,18 @@
-import React, {
-  Component
-} from 'react';
-// import logo from './logo.svg';
+import React, { Component } from 'react';
 import './styles/Main.scss';
 import TrailList from './TrailList.js';
-// import Header from './Header.js';
-// import Search from './Search.js';
 import LocationDisplay from './LocationDisplay.js';
 import LandingScreen from './LandingScreen.js';
 import Controls from './Controls.js';
```

Comment:
Dried up the css imports by using the main and removing the components we never used.